popup/popup.css

```
@@ -36,12 +36,41 @@ h2 {
   min-height: 20px;
   color: #1e293b;
 }
 
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)